src/node_buffer.cc

```diff
@@ -345,8 +345,8 @@ MaybeLocal<Object> New(Environment* env, size_t length) {
 
 
 MaybeLocal<Object> Copy(Isolate* isolate, const char* data, size_t length) {
+  EscapableHandleScope handle_scope(isolate);
   Environment* env = Environment::GetCurrent(isolate);
-  EscapableHandleScope handle_scope(env->isolate());
```

Comment:
Environment::GetCurrent() calls Isolate::GetCurrentContext() which creates a `Local<Context>` in the current handle scope, i.e., the scope of the caller.